### ISSUE TITLE
Caption for single runtime distributions in html pages

### ISSUE THREAD

**Issue by nikohansen** [numbbo/coco-postprocess#15]
The single function runtime distribution page `ppdata/.../pprldmany-single-functions/pprldmany.html` does not have a caption ([for example here](https://cma-es.github.io/lq-cma/ppdata-archives/pap-gecco2019/CMA-E_lq-CM_RANDO_SLSQP/pprldmany-single-functions/pprldmany.html#20)).

The root summary and function groups `pprldmany.html` page has a caption, but it doesn't explain the crosses or simulated restarts (to which "bootstrapped" refers):

> Bootstrapped empirical cumulative distribution of the number of objective function evaluations divided by dimension (FEvals/DIM) for 51 targets with target precision in $10^{[−8..2]}$ for all functions and subgroups in different dimensions. As reference algorithm, the best algorithm from BBOB 2009 is shown as light thick line with diamond markers.

One option, also to prevent code duplication, would be to provide longer descriptions on some documentation web page to which the captions link, rather than within the `cocopp` module?

**Comment by brockho** [comment]
> One option, also to prevent code duplication, would be to provide longer descriptions on some documentation web page to which the captions link, rather than within the `cocopp` module?

Agreed. But we should keep in mind that right now, some (if not all) captions are partly depending on the postprocessing setup (e.g. whether the `--expensive` setting is used). Not sure how to best deal with this.
